grpc/tests/src/test/system_contracts/auction/bids.rs

```diff
@@ -870,3 +872,245 @@ fn should_calculate_era_validators_multiple_new_bids() {
         BTreeSet::from_iter(vec![BID_ACCOUNT_1_PK, BID_ACCOUNT_2_PK,])
     );
 }
+
+#[ignore]
+#[test]
+fn undelegated_funds_should_be_released() {
+    // TODO: investigate why this is needed for use-system-contracts but not the host-side version
+    const SYSTEM_TRANSFER_AMOUNT: u64 = 1_000_000_000;
```

Comment:
Speaking of costs keep in mind that we account only for handful of opcodes right now: https://github.com/CasperLabs/casperlabs-node/blob/210b741d6251d6d10da552f05894946201caa0ae/execution_engine/src/shared/wasm_costs.rs#L41-L55 I plan to revisit it as part of EE-1040 (#279) where I still have to update the opcode table and wire it up to the wasmprep module. This might explain your concern - once we have all opcodes, gas, and host functions properly accounted this value should be increased. Given that I think we can remove this TODO

Reply:
Yes, I had started to arrive at this intuition after I wrote that.  Makes sense, will remove.